test/unit/util/audio-context.test.js

```diff
@@ -9,17 +13,14 @@ describe('Shared Audio Context', () => {
         expect(sharedAudioContext).toMatchObject({});
     });
 
-    test('returns AudioContext when mousedown is triggered', () => {
-        const sharedAudioContext = new SharedAudioContext();
-        const event = new Event('mousedown');
-        document.dispatchEvent(event);
-        expect(sharedAudioContext).toMatchObject(audioContext);
-    });
-
-    test('returns AudioContext when touchstart is triggered', () => {
-        const sharedAudioContext = new SharedAudioContext();
-        const event = new Event('touchstart');
-        document.dispatchEvent(event);
-        expect(sharedAudioContext).toMatchObject(audioContext);
+    // TODO: support both instead of either/or (see shared-audio-context)
+    // then make this two separate tests
+    test('returns AudioContext when mousedown/touchstart is triggered', () => {
+        const sharedAudioContext1 = new SharedAudioContext();
+        expect(sharedAudioContext1).toMatchObject({});
+        document.dispatchEvent(new Event('mousedown'));
+        document.dispatchEvent(new Event('touchstart'));
+        const sharedAudioContext2 = new SharedAudioContext();
```

Comment:
This test wasn't working before, but the Jest upgrade made it so that Jest now notices the failure 😅

The `SharedAudioContext` watches for either `mousedown` or `touchstart` (not both), depending on the characteristics of the browser, so one of the two tests should have failed in the previous configuration.

Also, `new SharedAudioContext()` grabs the value at the time of construction (function call) so neither test should have worked before.

Also also, we weren't resetting the state between tests, so the `mousedown` event could cause the `touchstart` test to pass or vice versa, depending on the order and timing of execution.

Ideally, `SharedAudioContext` should watch for both events, and we should reset the state (with `jest.resetModules()` or similar) in each test. That's extensive enough that I didn't want to make it part of this already-too-big PR.